Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using System.Linq;
+using UnityEngine;
+
+public class GameManager : MonoBehaviour
+{
+    public CardView cardPrefab;
+
+    List<Player> _players;
+    private Player _currentPlayer;
+
+    // GameManager singleton access
+    public static GameManager instance { get; private set; }
+    private void Awake() {
+        if (instance == null) {
+            instance = this;
+        }
+        else if (instance != this) {
+            Destroy(gameObject);
+        }
+
+        DontDestroyOnLoad(gameObject);
+    }
+
+	private void Start() {
+        // Init (players, scene, etc)
+        // Populate local player's hand

```

Comment:
Player.Start() currently handles shuffling the player's deck and drawing their beginning hand size.